src/main/java/nl/knaw/dans/bridge/plugin/lib/common/IResponseData.java

```diff
@@ -0,0 +1,28 @@
+package nl.knaw.dans.bridge.plugin.lib.common;
+
+import java.util.Optional;
+
+/*
+ * Created by Eko Indarto
+ */
+public interface IResponseData {
+
+    String getResponse();
+
+    default Optional<String> getState()  {
```

Comment:
Restricted to the ENUM, what does the bridge service expect here?

Reply:
👍

Reply:
Done!